README.md

```diff
@@ -22,12 +22,40 @@ Your goal is to produce shapes by cutting, rotating, merging and painting parts
 
 ## Building
 
--   Make sure `ffmpeg` is on your path
--   Install Node.js and Yarn
--   Install Java (required for textures)
--   Run `yarn` in the root folder
--   Cd into `gulp` folder
--   Run `yarn` and then `yarn gulp` - it should now open in your browser
+Note that this process is for Windows. Use equivalent actions for other operating systems if they are apparent. This guide is also meant to be understandable by users with little to no knowledge of Git.
+
+(credit to Hyperion-21)
+
+**SETUP**
+1. Download FFmpeg at https://www.gyan.dev/ffmpeg/builds/ffmpeg-release-full.7z
+2. Extract the .zip onto desktop, rename it to FFmpeg
+3. Move the folder into `C:\Program Files`, anywhere works but there is convenient
+4. Open the start menu and search for "env". Open the first result.
+5. Click on the listing that says `PATH` and click edit. 
+- On win10, you should see a list. Click `New` then type in `C:\Program Files\FFmpeg\bin`
+- On win7, you should see a window that says `Edit User Variable.` In the text field that says "Variable value:" you want to type at the end `;C:\Program Files\FFmpeg\bin`. The semicolon is for seperating the file path from the other file paths in that field.
+6. FFmpeg should now be installed. To test if you did this correctly, run `cmd.exe` as administrator and type `ffmpeg -version` and it should spit at you several lines of code.
+7. Install `Node.js`, `Yarn`, and `GitHub CLI`. All three of these softwares use setup wizards, so installation should be easy. If you don't know what setting to put in, use the default. Also make sure you have `Java` and `Git`, but both are very commom afaik. Run `cmd.exe` if you closed it earlier and run `java -version` and `git --version` to test if you already have them installed or not.
+8. Run `gh auth login` in `cmd.exe`. Login, selecting HTTP over SSH.
+9. `cd %USERPROFILE%` to return to your user folder.
+10. It is reccomended you create a new folder within your user folder for housing repos, but this is completely optional. Running `mkdir [name]` creates a new folder without having to open a Windows Explorer window. `cd` into it.
+11. `gh repo clone tobspr/shapez.io` to make sure the game's code is on your system. If you want to clone it into a different folder, `cd` into it. Personally,
```

Comment:
the folder thing is mentioned in the previous line, and "Personally," abruptly ends